### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TraderStf**
ParseHub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
    * https://redislabs.com/redis-cloud - Redis as a Service (25 mb free)
    * https://www.backand.com/ - Back-end as a service (for AngularJS)
    * http://www.zenginehq.com - Build business workflow apps in minutes - free for single users
+   * https://parsehub.com/ — Extract data from dynamic sites, turn dynamic websites into APIs.
```
Comment (ripienaar) [inline README.md]: please mention about its free things, just like "5 private repos" or whatever applies

Comment (TraderStf) [inline README.md]: ok for me
But which one, they are lot of possibilities between free and paying: Size, Number, User, Speed, Features?

Comment (ripienaar) [inline README.md]: just some quick note like the other entries here

**Comment by ripienaar** [comment]
sweet thanks

**Pull request merged**